package.json

```diff
@@ -62,11 +62,12 @@
   },
   "peerDependencies": {
     "react": "^0.14.9 || ^15.3.0",
-    "prop-types": "^15.5.0",
+    "prop-types": "^15.5.7",
     "react-redux": "^4.0.0 || ^5.0.0",
     "redux": "^3.5.2"
   },
   "dependencies": {
+    "prop-types": "^15.5.7",
```

Comment:
Thanks. Except the commit for the 3rd point (or I missed it)? :)

We need it also [here](https://github.com/gaearon/redux-devtools/blob/master/examples/counter/package.json) and [here](https://github.com/gaearon/redux-devtools/blob/master/examples/todomvc/package.json).